src/com/javabasic/lesson05/homework/Task02.java

```diff
@@ -0,0 +1,50 @@
+package com.javabasic.lesson05.homework;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Task02 {
+
+	static final int MAX_HEIGHT_BAG = 40;
+	static final int MAX_WIDTH_BAG = 55;
+	static final int MAX_DEPTH_BAG = 20;
+
+	static final int MAX_HEIGHT_CASE = 53;
+	static final int MAX_WIDTH_CASE = 40;
+	static final int MAX_DEPTH_CASE = 22;
+
+	static Integer heightBaggage;
+	static Integer widthBaggage;
+	static Integer depthBaggage;
+
+	public static void main(String[] args) throws IOException {
+		setSizeBaggage();
+		getTypeBaggage();
+	}
+
+	static void getTypeBaggage() {
```

Comment:
`get` wording is confusing, because method doesn't return anything. 
Better to say that this method printSomething instead get.